style.css

```diff
@@ -15,6 +15,8 @@ header {
     justify-content: flex-start;
     border: solid 1px darkgreen;
     width: 290px;;
+
+    
```

Comment:
Here, kindly remove the extra empty lines. You can check for this with linters. Your project should not have any linter errors. You can check for linters by following these instructions:

https://github.com/microverseinc/linters-config

Reply:
- [ ] Please, explain why this is important. 🐶